sippy-ng/src/chat/store/promptsSlice.js

```diff
@@ -0,0 +1,64 @@
+/**
+ * Zustand slice for managing slash command prompts
+ */
+export const createPromptsSlice = (set, get) => ({
+  // State
+  prompts: [],
+  promptsLoading: false,
+  promptsError: null,
+
+  // Fetch prompts from the server
+  fetchPrompts: async () => {
+    set({ promptsLoading: true, promptsError: null })
+
+    try {
+      const response = await fetch(
+        (process.env.REACT_APP_CHAT_API_URL || '/api/chat') + '/prompts'
```

Comment:
I'd like to see better url management. Can we put helper functions for all these URLS in a common place?